### DISCUSSION TITLE
Node deta .put doesn't work on larger files

### DISCUSSION THREAD

**Discussion by janzheng** [deta/cloud-docs#291] [Help] [answered]
Hi there,

I'm following the node tutorial (https://docs.deta.sh/docs/drive/node_tutorial) and realized `drive.put()` doesn't work for larger files — do you have to switch to chunked files and the HTTP API instead of using the node package for larger files?

Could you guys please add chunked file upload support to the node package?

Thanks!

**Comment by aavshr** [comment, marked as answer]
Hello @janzheng `drive.put()` already uses the chunked files API by default. The problem with the larger files might most likely be because of a Micro's `5.5 MB` maximum payload limit (https://docs.deta.sh/docs/micros/about#technical-specifications). Can you please tell what issue you get exactly?
